.gitignore

```diff
@@ -30,3 +30,4 @@ nbproject/
 # Eclipse
 .project
 .settings/
+
```

Comment:
Please remove this change. We don't want to have unnecessary changes in the PRs.